testutils/src/main/java/com/microsoft/identity/internal/testutils/labutils/policyHelper.java

```diff
@@ -0,0 +1,36 @@
+package com.microsoft.identity.internal.testutils.labutils;
+
+import com.microsoft.identity.internal.test.labapi.ApiException;
+import com.microsoft.identity.internal.test.labapi.api.DisablePolicyApi;
+import com.microsoft.identity.internal.test.labapi.api.EnablePolicyApi;
+import com.microsoft.identity.internal.test.labapi.model.CustomSuccessResponse;
+
+public class policyHelper {
+    public boolean enablePolicy(String upn, String policy)
+    {
+        LabAuthenticationHelper.getInstance().setupApiClientWithAccessToken();
+        final EnablePolicyApi enablePolicyApi = new EnablePolicyApi();
+        try{
+            final CustomSuccessResponse customSuccessResponse;
+            customSuccessResponse = enablePolicyApi.putPolicy(upn, policy);
+            final String expectedResult = ("Enabled : " + policy + "policy for user : " + upn)
+                    .toLowerCase();
+            return customSuccessResponse.getResult().toLowerCase().contains(expectedResult);
+        } catch (ApiException e) {
+            throw new RuntimeException(e.getMessage());
```

Comment:
Add logging

Reply:
Preferably, don't just crash. If we can recover from this, we should

Reply:
I think we do need to fail the test if this fails.....we can throw an `AssertionError` instead to just fail the test case.

Reply:
Added logging